### ISSUE TITLE
Add details to Git permissions request

### ISSUE THREAD

**Issue by jwulf** [Magikcraft/product-board#50]
Luigi writes in:

> I tried to log in but it asks me to give access to all the information about my contacts with read and also write permission. Why does it need write permission ?

So we can put a little link that says: "What permissions do we ask for, and why?" before they get asked for GitHub permissions.

**Comment by jwulf** [comment]
Jeremy writes:

> That looks very cool. I started the signin, but it was a bit too assertive for me. I don’t know much about it yet and it asks for a full integration with my GitHub account including read/write access to my gists. I cancelled, but I’ll do some more research and then try again later.

**Comment by tim-marwick** [comment]
Yep, I mentioned that before. Personally, I think we should do a much lighter weight sign-up.
The GitHub email verification step is also a pain for new users.

Of course, to go lighter weight, we would have to make GitHub integration optional, so it's a trade-off. But I think making registration as easy as possible is worthwhile.

**Comment by tim-marwick** [comment]
We don't currently capture the login redirect step in the PlayApp, which is actually the top of this funnel. So we don't know what drop-off is actually occurring there, but I agree, it maybe significant.

**Comment by jwulf** [comment]
Let's add this text to the current funnel:

```
Why do we ask for this? >

We create a public repo for your Magikcraft code, and synchronise the spells from your spellbook with it. 
You can see the repo that gets created [here](https://github.com/search?utf8=%E2%9C%93&q=magikcraft-spells&type=).
```
